### ISSUE TITLE
[Feature Request] Allow dynamic placement of panel

### ISSUE THREAD

**Issue by ThomasCox42** [erversteeg/Xp-Goals#7]
This plugin tool is amazing, but would be great is players could hold alt to move the panel around on the screen to wherever they want, similar to many other plugin panels. This would allow it to anchor to dynamic points, like the top left, so that it won't cover up other panels that are on screen. Just a suggestion for a future feature!

**Comment by erversteeg** [comment]
Yeah this is a great suggestion I will try and do some research on how I can get it switched over. I wasn't aware of holding down ult so I write in an anchor and position setting for that very reason.
